### PR TITLE
Fix User > Profile showing {getProgressDisplay()} - fixes #8412

### LINKED ISSUE #8412: Check-in counter at User > Profile is broken: next{{::getProgressDisplay()}}
As reported by RandomGryffindor in the Reprt a Bug guild:

Now at the User>Profile where you can see your check-in counter, it says: Progress until next{{::getProgressDisplay()}}.

image

### PR DESCRIPTION
**Pull request by gsacavdm**
Fixes #8412 

### Changes
We first attempt to use currentLoginDay.nextRewardAt and then do a null check on currentLoginDay.
Switch the sequence of those instructions to avoid error. 

----
UUID: 5783fd4a-dac9-466e-afea-15134a7365d9

### CODE REVIEW

**Comment by TheHollidayInn** [comment]
Looks good to me!

File: website/client-old/js/controllers/userCtrl.js [hunk 1]
```diff
@@ -93,8 +93,8 @@ habitrpg.controller("UserCtrl", ['$rootScope', '$scope', '$location', 'User', '$
 
     $scope.getProgressDisplay = function () {
       var currentLoginDay = Content.loginIncentives[$scope.profile.loginIncentives];
-      var nextRewardAt = currentLoginDay.nextRewardAt;
       if (!currentLoginDay || !nextRewardAt) return env.t('moreIncentivesComingSoon');
```
Comment (SabreCat) [inline website/client-old/js/controllers/userCtrl.js]: Need to remove the check on `!nextRewardAt` for this to work.

Comment (TheHollidayInn) [reply website/client-old/js/controllers/userCtrl.js]: Oh, wow. I definitely missed that.

Comment (gsacavdm) [reply website/client-old/js/controllers/userCtrl.js]: Yeah, I also missed that, my bad. Fixed!

**Comment by TheHollidayInn** [comment]
Noted towards your tier. Congrats on Tier 1 and thank you!

**Pull request merged**